### DISCUSSION TITLE
input value not showing

### DISCUSSION THREAD

**Discussion by renas-ghazi** [livewire/livewire#5147] [1. Help]
livewire 2.10

<input wire:model.defer='email' type="email" value"here is now showing ">

(when remove wire:model.defer value is working)

**Comment by Martleby** [comment]
I think this is the expected behaviour of '.defer'. What defer does is to prevent the server property being set/changed until an action on the page occurs that triggers a round trip to the server.

Here's a quick example playground: https://laravelplayground.com/#/snippets/9b7bb3a9-6bbe-41f4-8f78-67e88e7d8642

You might also want to checkout .debounce and .lazy for some other options.

Also, when using wire:model you don't need to (and probably shouldn't) set the value on the widget yourself, as Livewire does this for you.

HTH,
Martin.

**Comment by joshhanley** [comment]
@renas-ghazi yeah that is correct, you can't set the value of an input using the `value=""` attribute as Livewire is controlling the input.

To set the value, you should set your `$email` property inside your component class to whatever value should be displayed, and then due to the data-binding, Livewire will populate the input field for you.

```php
public $email;

public function mount()
{
    $this->email = 'my-email-value';
}
```

Hope this helps!
